ddtrace/profiling/collector/stack.pyx

```diff
@@ -2,7 +2,7 @@
 from __future__ import absolute_import
 
 import sys
-import threading
+import threading as ddtrace_threading
```

Comment:
The changes in this module are probably harder to justify without the full context of the referenced PR. So I'm thinking we can leave this source out?